app/Http/Controllers/Legacy/Web/ActionsController.php

```diff
@@ -133,11 +133,7 @@ public function update(Request $request, Action $action)
      */
     public function destroy(Action $action)
     {
-        $trashed = $action->delete();
-
-        if (! $trashed) {
-            return response()->json(['code' => 500, 'message' => 'There was an error deleting the action']);
-        }
+        $action->forceDelete();
```

Comment:
Not a big deal, but it might make sense to remove [the `SoftDeletes` trait](https://github.com/DoSomething/rogue/blob/b3219d7047dc69b389cdddb794c938ee0e7fe6a3/app/Models/Action.php#L10) and the `deleted_at` column from this model if we're not relying on soft deletion anymore.

Reply:
@DFurnes ah thanks for catching that! Made a PR for that [here](https://github.com/DoSomething/rogue/pull/865).